config/initializers/sidekiq.rb

```diff
@@ -15,6 +15,8 @@
     config.redis = ThreeScale::RedisConfig.new(System::Application.config.sidekiq).config
     config.error_handlers << System::ErrorReporting.method(:report_error)
 
+    config.logger.formatter = Sidekiq::Logger::Formatters::Pretty.new
```

Comment:
What was it beforehand?

Reply:
In Sidekiq 5 it was `Sidekiq::Logging::Pretty`, but they removed the class: https://github.com/sidekiq/sidekiq/wiki/Logging#api-changes

In Sidekiq 6 it took `System::ErrorReporting::LogFormatter` by default. Which I think it's wrong, and the logs where very cluttered.